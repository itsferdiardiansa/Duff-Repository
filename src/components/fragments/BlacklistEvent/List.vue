<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:blacklist="props">
          <div>
            <h3 class="font-black textr-lg mb-2">{{ props.data.title }}</h3>

            <p>
              blacklist at: <span>{{ props.data.blacklistAt }}</span>
            </p>
            <p>
              reported at: <span>{{ props.data.reportedAt }}</span>
            </p>
          </div>
        </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="warning">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Blacklist Event Name / Desc / TNC',
        accessor: 'blacklist',
        width: '40%',
        align: 'left',
      },
      {
        title: 'Created Date',
        accessor: 'createdDate',
        width: '60%',
        align: 'left',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.blacklistEvent.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
