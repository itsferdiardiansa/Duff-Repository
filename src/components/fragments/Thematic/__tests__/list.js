import { mount } from '@vue/test-utils';
import List from '@fragment/Thematic/List';
import thematicModule from '@store/modules/thematicPage';
import { createStore } from 'vuex';

describe('fragment/Thematic/List', () => {
  let store;

  beforeAll(() => {
    store = createStore({
      modules: {
        thematicPage: thematicModule,
      },
    });
  });

  it('will render and match snapshot', async () => {
    const Component = mount(<List />, {
      global: {
        plugins: [store],
      },
    });

    await Component.vm.$store.dispatch('thematicPage/fetchData', {
      page: 1,
      limit: 10,
    });
    expect(Component.html()).toMatchSnapshot();
  });
});
