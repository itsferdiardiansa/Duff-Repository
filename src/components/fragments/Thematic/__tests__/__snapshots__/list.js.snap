// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fragment/Thematic/List will render and match snapshot 1`] = `
<div class="list">
  <div class="ss-table-filter">
    <div class="ss-table-filter--top">
      <div class="ss-table-filter--input-form">
        <form class="ss-form search-form">
          <div class="ss-control--input has-icon bg-gray-100"><span class="icon"><svg xmlns="http://www.w3.org/2000/svg" class="ss-form-control--icon" viewBox="0 0 512 512"><path fill="currentColor" d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"></path></svg></span><input class="ss-input bg-gray-100" type="search" placeholder="Search" autocomplete="off">
            <!--v-if-->
            <!--v-if-->
          </div>
        </form>
      </div>
      <div class="ss-table-filter--extend"><button class="ss-button ss-button--orange ss-button--base text-bold">
          <div class="ss-button--wrapper">
            <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 448 512">
                <path fill="currentColor" d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"></path>
              </svg></div>
            <div class="ss-button--label has-icon">Create Thematic Page</div>
          </div>
        </button></div>
    </div>
    <div class="ss-table-filter--bottom">
      <div class="ss-table-filter--category"><label class="ss-table-filter--title">Sort By</label>
        <div class="w-40 ss-control--input w-40">
          <div class="ss-select">
            <div class="ss-select--button" tabindex="0"><span class="button-label">Newest</span><span class="button-icon"><svg xmlns="http://www.w3.org/2000/svg" class="h-4" viewBox="0 0 512 512"><path fill="currentColor" d="M8 256C8 119 119 8 256 8s248 111 248 248-111 248-248 248S8 393 8 256zm231-113.9L103.5 277.6c-9.4 9.4-9.4 24.6 0 33.9l17 17c9.4 9.4 24.6 9.4 33.9 0L256 226.9l101.6 101.6c9.4 9.4 24.6 9.4 33.9 0l17-17c9.4-9.4 9.4-24.6 0-33.9L273 142.1c-9.4-9.4-24.6-9.4-34 0z"></path></svg></span></div>
            <div class="ss-select--dropdown slide-top-enter-from slide-top-enter-active" style="display: none;">
              <ul class="dropdown-list">
                <li class="dropdown-item">Newest</li>
                <li class="dropdown-item">Oldest</li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <table class="ss-table">
    <thead class="ss-table--head" headers="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]" items="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]" isfetching="false" pagination="[object Object]" emptymessage="Empty data" rownumber="true" rowloader="5" selectablerows="false" withpagination="true" withfilter="true" filteroptions="">
      <tr>
        <th width="4%">
          <!--v-if--><span>#</span>
        </th>
        <th class="" width="10%">Banner</th>
        <th class="" width="10%" align="center">Banner Mobile</th>
        <th class="" width="15%" align="center">Thematic Name</th>
        <th class="">Page Info</th>
        <th class="" width="11%" align="center">Path</th>
        <th class="" align="center">Created At</th>
        <th class="" align="center"></th>
      </tr>
    </thead>
    <tbody class="ss-table--body">
      <!--v-if-->
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>1</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src="https://loket-development.s3.amazonaws.com/338f1e30-3dcc-430f-8b30-1a942042c1e1"></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src="https://loket-development.s3.amazonaws.com/a115b28e-a924-4735-bb5d-55ef3da1a546"></td>
        <td class="ss-table--body-col text-center">
          <div>thematic</div>
        </td>
        <td class="ss-table--body-col">
          <div>asadsd</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-03-03T06:19:20Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>2</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>Create a thematic page</div>
        </td>
        <td class="ss-table--body-col">
          <div>This is meta title has been edited</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>This is path</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-26T07:52:02Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>3</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src="https://loket-development.s3.amazonaws.com/98828b83-dbbb-4a0d-8260-d3bcd905a6c8"></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src="https://loket-development.s3.amazonaws.com/0e82debf-b60d-4bc8-a337-105f8707c078"></td>
        <td class="ss-table--body-col text-center">
          <div>thematic updateed</div>
        </td>
        <td class="ss-table--body-col">
          <div>updated</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>updat/sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-18T11:06:52Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>4</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>thematic neww</div>
        </td>
        <td class="ss-table--body-col">
          <div>asadsd</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-18T11:05:42Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>5</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>thematic</div>
        </td>
        <td class="ss-table--body-col">
          <div>asadsd</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-18T11:04:44Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>6</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>thematic</div>
        </td>
        <td class="ss-table--body-col">
          <div>asadsd</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-18T11:03:48Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>7</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>thematic</div>
        </td>
        <td class="ss-table--body-col">
          <div>asadsd</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-18T11:00:50Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>8</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src="iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYAAAD0eNT6AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEwAACxMBAJqcGAAAIABJREFUeJzt3XmcnHWZ9/vr+t3d6UTCYjpiIiJBPS6gcXAbkUdlkBkWdVx42c7yuGQEc4Z0VXUWBlSGssEFDkm6ayFzWpSg+OghI+goGEQElUUFFQVcHmU0iEKQdKMkkF6qruv8QesDCiHJ/au6a/m8/9LXy/r"></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src="iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYAAAD0eNT6AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEwAACxMBAJqcGAAAIABJREFUeJzt3XmcnHWZ9/vr+t3d6UTCYjpiIiJBPS6gcXAbkUdlkBkWdVx42c7yuGQEc4Z0VXUWBlSGssEFDkm6ayFzWpSg+OghI+goGEQElUUFFQVcHmU0iEKQdKMkkF6qruv8QesDCiHJ/au6a/m8/9LXy/r"></td>
        <td class="ss-table--body-col text-center">
          <div>thematic</div>
        </td>
        <td class="ss-table--body-col">
          <div>asadsd</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>sad</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-18T10:58:48Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>9</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>Thematic name</div>
        </td>
        <td class="ss-table--body-col">
          <div>Meta title</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>Path</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-15T02:45:05Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
      <tr class="ss-table--body-row">
        <td class="ss-table--body-col"><label>10</label></td>
        <td class="ss-table--body-col"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center"><img class="h-10 inline-block" src=""></td>
        <td class="ss-table--body-col text-center">
          <div>Add new thematic</div>
        </td>
        <td class="ss-table--body-col">
          <div>Meta title</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>https://youtube.com</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div>2021-02-15T02:42:39Z</div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper">
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 512 512">
                  <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                </svg><span>Edit</span></div>
              <div class="btn"><svg xmlns="http://www.w3.org/2000/svg" class="w-4" viewBox="0 0 448 512">
                  <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                </svg><span>Delete</span></div>
            </div>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
  <div class="ss-pagination">
    <!--v-if-->
    <div class="ss-pagination--page"><span class="page-number active">1</span><span class="page-number">2</span><span class="nav-btn next" type="nav"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 320 512"><path fill="currentColor" d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"></path></svg></span></div>
  </div>
</div>
<!--v-if-->
`;
